packages/backend/src/routers/projectRouter.ts

```diff
@@ -182,6 +182,39 @@ projectRouter.post(
                 metricQuery,
                 req.params.projectUuid,
                 req.params.exploreId,
+                false,
+            );
+            res.json({
+                status: 'ok',
+                results,
+            });
+        } catch (e) {
+            next(e);
+        }
+    },
+);
+
+projectRouter.post(
+    '/explores/:exploreId/runQueryAll',
+    isAuthenticated,
+    async (req, res, next) => {
+        try {
+            const { body } = req;
+            const metricQuery: MetricQuery = {
+                dimensions: body.dimensions,
+                metrics: body.metrics,
+                filters: body.filters,
+                sorts: body.sorts,
+                limit: 1000000,
```

Comment:
so there's still a limit, should we let users know the amount?

Reply:
Yeah, this is something I'm not sure about... changing the limit to `undefined` will imply a few extra changes in the methods we use to run queries. In most DB the limit is measures in MB apparently. I found this on Stackoverflow: `MySQL suggests using 18446744073709551615 as the number of records in the limit` it is probably a maxInt or something ... but I don't think that limit is compatible with all databases... so I don't know, maybe 1000M ?

Reply:
I'm going to do a refactor to specify an undefined limit